decentwork/apps/professions/serializers.py

```diff
@@ -0,0 +1,11 @@
+from rest_framework import serializers
+
```

Comment:
Useless line?

Reply:
Nah rest_framework is a third-party app and the next import is from our app so there is one line break between third-party and our app imports.